control.yml

```diff
@@ -1,3 +1,4 @@
 - import_playbook: "{{ playbook_dir }}/deploy-with-{{ provider | default('lago') }}.yml"
 - import_playbook: "{{ playbook_dir }}/deploy-{{ cluster_type | default('openshift') }}.yml"
-- import_playbook: "{{ playbook_dir }}/install-kubevirt-release.yml"
```

Comment:
I think you can remove `install-kubevirt-release.yml` playbook then.

Reply:
Agreed, I'll include it in this pr